### PR TITLE
Add Feedback UI and FeedbackServlet

### PR DESCRIPTION
**Pull request by kaylaleung**
Open to suggestions about how I can maybe split up this PR :')

It's just hard to see how the feedback is changing without the UI.

Also JUnit tests fail onces Datastore requests are involved because it doesn't know where to read database content from. Will need another issue/PR to address that.

### CODE REVIEW

File: src/main/java/com/google/step/snippet/external/W3SchoolsClient.java [hunk 1]
```diff
@@ -11,6 +12,8 @@
   private static final String DESC_TAG = "p";
   private static final String SNIPPET_CLASS = "w3-example";
   private static final String CODE_CLASS = "w3-code";
+  private static final String UP = "upvote";
+  private static final String DOWN = "downvote";
```
Comment (irisliu77) [inline src/main/java/com/google/step/snippet/external/W3SchoolsClient.java:16]: you can define these to parameters in the interface

File: src/main/java/com/google/step/snippet/servlets/FeedbackServlet.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  private static final String URL = "url";
+  private static final String FEEDBACK = "feedback";
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Entity feedbackEntity = null;
+    String url = request.getParameter(URL);
+
+    if (url == null) {
+      System.out.println("Invalid Card");
```
Comment (irisliu77) [inline src/main/java/com/google/step/snippet/servlets/FeedbackServlet.java:28]: remove print statement

File: src/main/java/com/google/step/snippet/servlets/FeedbackServlet.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    long totalVotes = 0;
+
+    if (feedbackEntity == null) {
+      feedbackEntity = new Entity(FEEDBACK);
+      feedbackEntity.setProperty(URL, url);
+      /* Initialize vote count */
+      if (request.getParameter(UP) != null) {
+        totalVotes = 1;
+        feedbackEntity.setProperty(UP, (long) 1);
+        feedbackEntity.setProperty(DOWN, (long) 0);
```
Comment (irisliu77) [inline src/main/java/com/google/step/snippet/servlets/FeedbackServlet.java:46]: i don't see the necessity of casting long type here

File: src/main/webapp/css/srp_style.css [hunk 4]
```diff
@@ -17,24 +17,36 @@
...
+.card .content .card-description {
   font: bold 16px Roboto, Arial, sans-serif;
   margin-top: 20px;
 }
 
-.card code {
+.card .content code {
   margin-top: 10px;
   margin-bottom: 10px;
   display: block;
```
Comment (irisliu77) [inline src/main/webapp/css/srp_style.css:35]: i think it looks better when display inline

**Comment by kaylaleung** [comment]
Will close this for now to separate out front end and backend code

**Pull request closed without merging**